src/main/java/org/owasp/esapi/reference/DefaultEncoder.java

```diff
@@ -349,12 +354,18 @@ public String encodeForDN(String input) {
         if ((input.length() > 0) && ((input.charAt(0) == ' ') || (input.charAt(0) == '#'))) {
             sb.append('\\'); // add the leading backslash if needed
         }
+        // According to [1] and [2], the encoder should escape forward slash ('/') in DNs.
```

Comment:
@noloader - Not a big deal, but we probably don't need this same comment in here multiple times. I'd recommend removing this one. The comment on lines 308-310 should be sufficient.

Reply:
v2 removes the second comment.